### ISSUE TITLE
Make sign in page fit on one screen in mobile

### ISSUE THREAD

**Issue by PepperTeasdale** [SandersForPresident/BernieBNB#264]
We would like the sign in page to fit on one screen and have a little padding at the bottom still (currently the sign in with google button is at the very bottom of the page).

We haven't firmed up the best way to do this, but we could make the logo smaller and make the instructions more simple/ shorter.

**Comment by jasnow** [comment]
@cockroachbill - Are you still going to work this issue?

**Comment by cockroachbill** [comment]
Yes, sorry I dropped off for a while there. I'll get what I have on this pushed. I was also working on optimizing the rest of the pages for mobile

**Comment by jasnow** [comment]
Thanks

**Comment by jasnow** [comment]
@cockroachbill - How's the progress on this issue?
